libraries/freertos_plus/standard/https/include/iot_https_client.h

```diff
@@ -344,6 +409,131 @@ IotHttpsReturnCode_t IotHttpsClient_SendSync(IotHttpsConnectionHandle_t connHand
                                              uint32_t timeoutMs);
 /* @[declare_https_client_sendsync] */
 
+/**
+ * @brief Asynchronous send of the the HTTPS request.
+ * 
+ * This function will invoke each of the non-NULL callbacks configured in #IotHttpsAsyncInfo_t.callbacks when the
```

Comment:
Strictly speaking, it won't invoke each of the callback, it will invoke the appropriate, or as needed callbacks.

Reply:
Thanks I updated.